plugins/custom-cols.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-cols
+spec:
+  version: "v0.0.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/webofmars/kubectl-custom-cols/archive/v0.0.2.tar.gz
+    sha256: "f01d1aa63c320989071baa88d7b0a8fefa1147e0da8b9edf32874def21c84736"
+    files:
+    - from: "./kubectl-custom-cols-0.0.2/kubectl-custom-cols"
+      to: "."
+    - from: "./kubectl-custom-cols-0.0.2/templates/*.tpl"
+      to: "templates"
+    bin: "./kubectl-custom-cols"
+  shortDescription: Easy use of custom columns output formating.
+  homepage: https://github.com/webofmars/kubectl-custom-cols
+  caveats: |
+    For now you can't list witch templates are available and adding some can only
+    be done either by doing a PR upstream (see github repo) or by copying a file in 
+    krew store (urgllll).
+  description: |
+    This plugin allows you to pre-define some custom columns file templates that
```

Comment:
It seems like this is a "kubectl get" replacement, but it doesn't say so anywhere (incl. the tool's own repository).

I'll leave it for you to decide but a better shortDescription could be:

> A "kubectl get" replacement with customizable column presets.

Reply:
Nice one thxs